examples/ios-device.ts

```diff
@@ -15,11 +15,16 @@ import { parseAppMeta, pause, platformApi } from '../src/index';
     });
 
     const appPath = process.argv[3] || '/path/to/app-files';
-    const proxyHost = process.argv[4];
-    const proxyPort = process.argv[5];
+    const caCertPath = process.argv[4];
+    const proxyHost = process.argv[5];
+    const proxyPort = process.argv[6];
 
     await ios.ensureDevice();
 
+    if (caCertPath) {
+        await ios.removeCertificateAuthority(caCertPath);
+        await ios.installCertificateAuthority(caCertPath);
```

Comment:
I think we should maybe include a prompt or something waiting for you to manually accept the cert once to emphasize in the example that this is a manual task currently.

Reply:
I like the idea but it would be annoying to always have to confirm that prompt in the example (especially for us, I use those for testing). I would much rather invest the time needed to do this properly into actually fixing the problem or at least being able to check whether the user has done that once.

As a compromise, I'll add a `console.log()`.